src/sentry/ingest/ingest_consumer.py

```diff
@@ -114,6 +114,11 @@ def shutdown(self):
 
 @metrics.wraps("ingest_consumer.process_transactions_batch")
 def process_transactions_batch(messages, projects):
+    if options.get("sentry:ingest-transactions-celery") == "1":
```

Comment:
I think we should add a proper (one that we register) option here, because it will be very easy to make a change and then forget about that.
Also, an explicit comment would help here.

Reply:
👍 Added the explicit option and documented it.